tests/components/mqtt/test_subscription.py

```diff
@@ -179,10 +176,10 @@ def msg_callback(*args):
         sub_state,
         {"test_topic1": {"topic": "test-topic1", "msg_callback": msg_callback}},
     )
-    call_count = mock_mqtt.async_subscribe.call_count
+    call_count = mqtt_mock.async_subscribe.call_count
     sub_state = await async_subscribe_topics(
         hass,
         sub_state,
         {"test_topic1": {"topic": "test-topic1", "msg_callback": msg_callback}},
     )
-    assert call_count == mock_mqtt.async_subscribe.call_count
+    assert call_count == mqtt_mock.async_subscribe.call_count
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5737505)